### ISSUE TITLE
gpu bugging out and showing 511 degrees celcius

### ISSUE THREAD

**Issue by DeJamesBong** [fireice-uk/xmr-stak-amd#168]
i have a vega 56 in my gaming computer alongside a 980ti, had this mining fine for several days without issues (apart from hbm2 memory would drop out now and then), switched off pc and moved location, upon switching back on i can now only mine for about an hour at most until wattman and gpu-z start showing 511 degrees and causes system to freeze (have to restart system every time), what causes this sensor bug and whats the best resolution to resolve this?

TIA Jimmy B

**Comment by Grimm2017** [comment]
And you are running xmr stak?  I don't see how this could be caused by xmr stak.

**Comment by DeJamesBong** [comment]
Yes I am running xmr stak. It has also happened when just on  the desktop just now,  so no it is not xmr stak causing this issue.  Thanks for reply Grimm. I guess this case can be closed now and off to scour Google even more for some answers.

**Comment by mat91** [comment]
Hi, i have the same problem with my vega 56.. 511 degrees and i have to restart the system manually. Have you found a solution? thank you

**Comment by jimboolio** [comment]
I have this issue too, but it occurs on windows and linux and I am not even sure if I do have xmr stack. It is most likely a issue with the gpu.
